src/main/java/crud/model/Driver.java

```diff
@@ -6,10 +6,14 @@
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
+    private String password;
 
-    public Driver(String name, String licenceNumber) {
+    public Driver(String name, String licenceNumber, String login, String password) {
```

Comment:
Better to add a new constructor, because you can use the old constructor in other places